portfolio/src/main/java/com/google/sps/data/Information.java

```diff
@@ -0,0 +1,85 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Information {
+    private long id;
+    private String email;
+    private ArrayList<String> type;
+    //private String type;
+    private String name;
+    private String comment;
+    private long timestamp;
+  
+  public Information()
```

Comment:
If this constructor isn't being used, you can probably just delete it.

Reply:
OK, I got it.